.docker/entrypoint.sh

```diff
@@ -56,6 +56,11 @@ function delib_clean () {
         cd $cwd
 }
 
+function qt_rendering() {
```

Comment:
consider naming this `qt_software_rendering` or something that indicates it's switching to software rendering.

Reply:
will do.  how about `qt_soft_render` just to keep shorter ?

Reply:
Sure!